client/pdo/client/controller/commands/contract.py

```diff
@@ -85,7 +86,7 @@ def load_contract(state, contract_file) :
         data_directory = state.get(['Contract', 'DataDirectory'])
         ledger_config = state.get(['Ledger'])
 
-        return Contract.read_from_file(ledger_config, contract_file, data_dir=data_directory)
+        return Contract.read_from_file(ledger_config, contract_file, data_dir=data_directory, interpreter=os.getenv('PDO_INTERPRETER'))
     except Exception as e :
```

Comment:
might be better to set this explicitly in the shell wrapper & then check the state for the value

Reply:
Done.

Reply:
Addressed in https://github.com/hyperledger-labs/private-data-objects/pull/286/commits/935716230bfc04a2cddb0e51207e579baea6ef39